homework-1/task_2.py

```diff
@@ -0,0 +1,44 @@
+class Student:
+
+    def __init__(self, name, age, id, subjects):
+        self.name = name
+        self.age = age
+        self.id = id
+        self.subjects = subjects
```

Comment:
Same as above. Better to define what subject is instead of letting users pass something in as your methods rely on the data structure of `self.subjects`